lib/services/device/device-install-app-service.ts

```diff
@@ -49,12 +48,38 @@ export class DeviceInstallAppService {
 			device,
 			projectDir: projectData.projectDir,
 		});
+		const buildOutputOptions = platformData.getValidBuildOutputData(buildData);
+		const outputPath = buildData.outputPath || platformData.getBuildOutputPath(buildData);
+		const packages = await this.$buildArtifactsService.getAllAppPackages(
+			outputPath,
+			buildOutputOptions
+		);
+		let packageFile;
+		if (packages.length === 1) {
+			// will always be the case on iOS
+			packageFile = packages[0].packageName;
```

Comment:
```suggestion
			packageFile = packages.at(0).packageName;
```

Reply:
@rigor789 not sure of the difference there, `packages` is just an array. Why is it better?

Reply:
It's the same - just reads nicer I guess. Doesn't matter actually... The main benefit is when accessing the last element for example `array.at(-1)` would return the last element...